src/Components/Server/src/Circuits/CircuitHost.cs

```diff
@@ -370,6 +451,43 @@ static Log()
                    LogLevel.Debug,
                    EventIds.OnCircuitClosed,
                    "Closing circuit with id {CircuitId}.");
+
+                _beginInvokeDotNetStatic =
+                    LoggerMessage.Define<string, string, string>(
+                    LogLevel.Debug,
+                    EventIds.BeginInvokeDotNet,
+                    "Invoking static method '[{Assembly}]::{MethodIdentifier}' with callback id '{CallId}'");
```

Comment:
This is a nitpick, but using this string format may be confusing because it looks like a CLR-style method identifier but isn't. How about:

```suggestion
                    "Invoking static method with identifier '{MethodIdentifier}' on assembly '{Assembly}' with callback id '{CallId}'");
```